### PR TITLE
Consistent formatting of references to headers and preferences

### PR DESCRIPTION
**Pull request by ralfhandl**
Consistently reference headers and preferences

### CODE REVIEW

File: odata-protocol/11.4 Data Modification.md [hunk 1]
```diff
@@ -394,7 +393,7 @@ Upon successful completion the service responds with either
 [`200 OK`](#ResponseCode200OK) and a representation of the updated
 entity, or [`204 No Content`](#ResponseCode204NoContent). The client may
 request that the response SHOULD include a body by specifying a
-[`Prefer` header](#Preferencereturnrepresentationandreturnminimal) with a value of
+[`Prefer`](#Preferencereturnrepresentationandreturnminimal) header with a value of
 [`return=representation`](#Preferencereturnrepresentationandreturnminimal), or by
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:397]:
```suggestion
[`return=representation`](#Preferencereturnrepresentationandreturnminimal) preference, or by
```

File: odata-protocol/11.4 Data Modification.md [hunk 2]
```diff
@@ -689,14 +688,14 @@ OData format supported by the service. It is not possible to set the
...
+Upon successful completion, the response MUST contain
+[`Location`](#HeaderLocation) header that contains the edit URL of the created media
 entity.
 
 Upon successful completion the service responds with either
 [`201 Created`](#ResponseCode201Created), or
 [`204 No Content`](#ResponseCode204NoContent) if the request included a
-[`Prefer` header](#Preferencereturnrepresentationandreturnminimal) with a value of
+[`Prefer`](#Preferencereturnrepresentationandreturnminimal) header with a value of
 [`return=minimal`](#Preferencereturnrepresentationandreturnminimal).
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:699]:
```suggestion
[`return=minimal`](#Preferencereturnrepresentationandreturnminimal) preference.
```

File: odata-protocol/11.4 Data Modification.md [hunk 3]
```diff
@@ -844,8 +843,8 @@ property.
 Upon successful completion the service responds with either
 [`200 OK`](#ResponseCode200OK) or
 [`204 No Content`](#ResponseCode204NoContent). The client may request
-that the response SHOULD include a body by specifying a [Prefer
-header](#Preferencereturnrepresentationandreturnminimal) with a value of
+that the response SHOULD include a body by specifying a
+[Prefer](#Preferencereturnrepresentationandreturnminimal) header with a value of
 [`return=representation`](#Preferencereturnrepresentationandreturnminimal).
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:848]:
```suggestion
[`return=representation`](#Preferencereturnrepresentationandreturnminimal) preference.
```

File: odata-protocol/11.4 Data Modification.md [hunk 4]
```diff
@@ -897,8 +896,8 @@ request body and set all unspecified properties to their default value.
...
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:901]:
```suggestion
[`return=representation`](#Preferencereturnrepresentationandreturnminimal) preference.
```

File: odata-protocol/11.4 Data Modification.md [hunk 5]
```diff
@@ -929,8 +928,8 @@ supported for collection properties.
 Upon successful completion the service responds with either
 [`200 OK`](#ResponseCode200OK) and a representation of the updated collection, or
 [`204 No Content`](#ResponseCode204NoContent). The client may request
-that the response SHOULD include a body by specifying a [Prefer
-header](#Preferencereturnrepresentationandreturnminimal) with a value of
+that the response SHOULD include a body by specifying a
+[Prefer](#Preferencereturnrepresentationandreturnminimal) header with a value of
 [`return=representation`](#Preferencereturnrepresentationandreturnminimal).
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:933]:
```suggestion
[`return=representation`](#Preferencereturnrepresentationandreturnminimal) preference.
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: odata-protocol/11 Data Service Requests.md [hunk 6]
```diff
@@ -1055,13 +1055,13 @@ If the resource path identifies a collection, the response MUST be the
 format-specific representation of a collection of entity references
 pointing to the related entities. If no entities are related, the
 response is the format-specific representation of an empty collection.
-The response MAY contain an [ETag header](#HeaderETag) for the
+The response MAY contain an [ETag](#HeaderETag) header for the
```
Comment (HeikoTheissen) [inline odata-protocol/11 Data Service Requests.md:1058]:
```suggestion
The response MAY contain an [`ETag`](#HeaderETag) header for the
```

File: odata-protocol/11 Data Service Requests.md [hunk 7]
```diff
@@ -1055,13 +1055,13 @@ If the resource path identifies a collection, the response MUST be the
...
 collection whose value changes if the collection of references changes,
 i.e. a reference is added or removed.
 
 If the resource path identifies a single existing entity, the response
 MUST be the format-specific representation of an entity reference. The
-response MAY contain an [ETag header](#HeaderETag) which represents the
+response MAY contain an [ETag](#HeaderETag) header which represents the
```
Comment (HeikoTheissen) [inline odata-protocol/11 Data Service Requests.md:1064]:
```suggestion
response MAY contain an [`ETag`](#HeaderETag) header which represents the
```

File: odata-protocol/11.4 Data Modification.md [hunk 8]
```diff
@@ -41,11 +41,11 @@ announce this via annotations with the terms
 [OData-VocCap](#ODataVocCap).
 
 If optimistic concurrency control is required for a resource, the
-service MUST include an [ETag header](#HeaderETag) in a response to a
+service MUST include an [ETag](#HeaderETag) header in a response to a
```
Comment (HeikoTheissen) [inline odata-protocol/11.4 Data Modification.md:44]:
```suggestion
service MUST include an [`ETag`](#HeaderETag) header in a response to a
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

**Comment by HeikoTheissen** [comment]
> Services SHOULD order language-dependent strings according to the [content-language](#HeaderContentLanguage) of the response

Should the "content-language" be in monospace? Or should the hyphen be deleted?

**Comment by HeikoTheissen** [comment]
> The request body MUST use the content-type `text/plain`.

Should we remove the hyphen here?

**Comment by ralfhandl** [comment]
> > The request body MUST use the content-type `text/plain`.
> 
> Should we remove the hyphen here?

Changed to 

> The request body MUST use `Content-Type: text/plain`.

**Pull request merged**